felix/fv/mtu_test.go

```diff
@@ -99,11 +99,12 @@ var _ = infrastructure.DatastoreDescribe("VXLAN topology before adding host IPs
 			}
 
 			// Disable VXLAN. We should expect the MTU to change, but still based on the default 1460.
-			fc, err := client.FelixConfigurations().Get(context.Background(), "default", options.GetOptions{})
```

Comment:
It might be worth a test to verify that MTU changes correctly when enabling / disabling VXLAN via an IP pool as well, since that should now work similarly

Reply:
Will do, thanks!